src/internal/cache-client.ts

```diff
@@ -462,12 +462,12 @@ export class CacheClient {
     value: string | Uint8Array,
     ttl: CollectionTtl = CollectionTtl.fromCacheTtl(),
     truncateBackToSize?: number
-  ): Promise<CacheListFetch.Response> {
+  ): Promise<CacheListPushFront.Response> {
```

Comment:
seems like this should've caused an integ test failure?

Reply:
Sure does!

I think typescript's type checking is a lot more lax than we'd like. `CacheListFetch.Response` and `CacheListPushFront.Response` implement the same methods, so maybe it saw them as equivalent?

Reply:
I took a quick glance at the integ tests for `ListPushFront` and I think the integ-test didn't fail because we're not asserting the actual type of the response, e.g. here https://github.com/momentohq/client-sdk-javascript/blob/77628ff08b9dbbed825e893f98a06387a09b2635/integration/list.test.ts#L56

Reply:
I'll add a test.